tests/fixtures/nwchem/references.py

```diff
@@ -67,3 +67,13 @@
         }
     ]
 }
+
+
+REFERENCE_VALUES = {
+    "7.0.0": {
+        "scf": {
+            "total_energy": TOTAL_ENERGY,
+            "total_energy_contributions": TOTAL_ENERGY_CONTRIBUTION,
+        }
+    }
+}
```

Comment:
(1) https://intellij-support.jetbrains.com/hc/en-us/community/posts/360001957340-Ensure-line-feed-at-file-end-on-Reformat
(2) should we keep references under git or git-lfs like in other folders?